src/fragments/lib/analytics/js/existing-resources.mdx

```diff
@@ -8,11 +8,25 @@ import { Amplify } from 'aws-amplify';
 Amplify.configure({
   ...Amplify.getConfig(),
   Analytics: {
+    ...Amplify.getConfig().Analytics,
     Pinpoint: {
       // REQUIRED -  Amazon Pinpoint App Client ID
       appId: 'XXXXXXXXXXabcdefghij1234567890ab',
+
       // REQUIRED -  Amazon service region
-      region: 'us-east-1'
+      region: 'us-east-1',
+
+      // OPTIONAL - The buffer size for events in number of items.
```

Comment:
```suggestion
      // OPTIONAL - How many events can be buffered at once.
```

Reply:
The other Analytics service providers are using the same description for these options. I can update all of them at once in another PR. 

ref: https://github.com/aws-amplify/docs/blob/next-release/main/src/fragments/lib/analytics/js/streaming.mdx#L35-L45